app/views/users/personas.html.erb

```diff
@@ -2,30 +2,17 @@
   <span class="govuk-caption-xl">
     Register trainee teacher
   </span>
-  Users
+  Personas
 </h1>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render_component Users::Persona::View.new(
-      name: "Adam Baker",
-      roles: ["Admin"],
-      description: '<p class="govuk-body">Belongs to Provider A and is responsible for managing users.</p>',
-      link_path: root_path
-    )%>
-
-    <%= render_component Users::Persona::View.new(
-      name: "Annie Bell",
-      roles: [],
-      description: '<p class="govuk-body">Belongs to Provider A and creates trainees for assessment only routes.</p>',
-      link_path: root_path
-    )%>
-
-    <%= render_component Users::Persona::View.new(
-      name: "Bridget Campbell",
-      roles: ["Admin"],
-      description: '<p class="govuk-body">Belongs to Provider B,responsible for managing users and creates trainees for assessment only routes.</p>',
-      link_path: root_path
-    )%>
+    <%- personas.each do |persona| %>
+      <%= render_component Users::Persona::View.new(
+        name: persona.name,
+        description: %(<p class="govuk-body">Belongs to <strong>#{persona.provider.name}</strong> and is responsible for managing users.</p>),
+        link_path: sessions_path(user_id: persona))
+      %>
+    <% end %>
```

Comment:
can we rework it so it conforms to https://viewcomponent.org/#rendering-collections

```suggestion
      <%= render_component Users::Persona::View.with_collection(@personas)) %>%>
```

Reply:
Can do.